recipes/libressl/2.9.2/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class PackageTest(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
+        self.copy("*.dll", "", "bin")
```

Comment:
Probably this is not needed when running with `run_environment=True`

Reply:
indeed